README.md

```diff
@@ -59,6 +59,18 @@ data:
   azure_region: Base64encodeRegion
 ```
 
+### Equinix Metal
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: kube-system
+  name: equinix-metal-credentials
+data:
+  PACKET_API_KEY: EquinixMetalAPIKey
```

Comment:
Seeing this, we should likely change the name of this variable to `EQUINIX_METAL_API_KEY` or similar. That would require changes to the cluster-api-provider-equinix-metal PR: https://github.com/openshift/cluster-api-provider-equinix-metal/pull/1#issuecomment-759025769 and installer PR: https://github.com/openshift/installer/pull/4472

Reply:
+1, i think that makes sense for clarity